ludwig/schema/metadata/encoder_metadata.py

```diff
@@ -12412,6 +12412,637 @@
             internal_only=True,
         ),
     },
+    "TVAlexNetEncoder": {
+        "model_variant": ParameterMetadata(
+            ui_display_name=None,
+            default_value_reasoning=None,
+            example_value=None,
+            related_parameters=None,
+            other_information=None,
+            description_implications=None,
+            suggested_values=None,
+            suggested_values_reasoning=None,
+            commonly_used=False,
+            expected_impact=ExpectedImpact.UNKNOWN,
+            literature_references=None,
+            internal_only=False,
```

Comment:
@jimthompson5802 This all looks good. The only attribute I would pay special attention to for now is `internal_only` - I'm guess the answer to this is no, but is there any parameter you've defined on any new schema that is kept around solely as a piece of internal Ludwig state and that *should not* be configurable by another Ludwig user? (If so, set that parameter's `internal_only` to `True`.)